### PR TITLE
Android: support multi-touch

### PR DESCRIPTION
**Pull request by maxded**
Small PR enabling multi-touch on Android :) 

Also rearranged an entry in the changelog as it was added to the top instead of the bottom.

- [x] Tested on all platforms changed
- [x] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] `cargo doc` builds successfully
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users

### CODE REVIEW

File: src/platform_impl/android/mod.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#![cfg(target_os = "android")]
+//#![cfg(target_os = "android")]
```
Comment (msiglreith) [inline src/platform_impl/android/mod.rs:1]: unintentional change?

File: src/platform_impl/android/mod.rs [hunk 2]
```diff
@@ -181,37 +181,45 @@ impl<T: 'static> EventLoop<T> {
...
+                                                y: pointer.y() as _,
                                             };
-                                            call_event_handler!(
-                                                event_handler,
-                                                self.window_target(),
-                                                control_flow,
-                                                event
-                                            );
+
+                                            if let Some(phase) = phase {
```
Comment (msiglreith) [inline src/platform_impl/android/mod.rs:203]: minor: could move the if condition out of the loop

**Review by msiglreith — COMMENTED** [review]
thanks!

File: src/platform_impl/android/mod.rs (hunk 1 above)
Comment (maxded) [reply src/platform_impl/android/mod.rs:1]: Oh whoops :)

**Comment by maxded** [comment]
Done, should be good to go now!

**Review by msiglreith — APPROVED** [review]

**Pull request merged**